interface/usergroup/facilities.php

```diff
@@ -1,7 +1,7 @@
 <?php
 require_once("../globals.php");
```

Comment:
Added this to codebase in following PR:
https://github.com/openemr/openemr/pull/1256

Do that because of the use of fancybox, needed to use another mechanism to bring in bootstrap to avoid jquery/bootstrap/fancybox js issues